config/config.go

```diff
@@ -118,6 +118,8 @@ type Config struct {
 	InnkeeperPreRouteFilters  string               `yaml:"innkeeper-pre-route-filters"`
 	InnkeeperPostRouteFilters string               `yaml:"innkeeper-post-route-filters"`
 	RoutesFile                string               `yaml:"routes-file"`
+	RemoteRoutesURLs          *listFlag            `yaml:"remote-routes-urls"`
```

Comment:
nitpick: url can be local, do we need the `remote-` prefix?

Reply:
My thinking was to group related flags with the same prefix (we introduce two) to avoid lonely flags that one can have trouble to connect to the specific feature. In this case `remote-` seems to be handy as that's the name of the feature. Do you have a better naming suggestion considering there are two flags?